api/admin/build.go

```diff
@@ -103,10 +104,11 @@ func AllBuildsQueue(c *gin.Context) {
 
 // UpdateBuild represents the API handler to update a build.
 func UpdateBuild(c *gin.Context) {
-	logrus.Info("Admin: updating build in database")
-
 	// capture middleware values
 	ctx := c.Request.Context()
+	l := c.MustGet("logger").(*logrus.Entry)
+
+	l.Debug("platform admin: updating build")
```

Comment:
consistency nitpick, some funcs had the logging moved to the top of the function (`AllBuildsQueue`) and some were moved below middleware values

Reply:
fwiw i prefer it being at the very top if it doesnt require anything from the middleware

Reply:
found a couple of spots. fixed in https://github.com/go-vela/server/pull/1145/commits/2508d5f5fd067e9b3510914c27d4322e20ca3e2e